features/customers.feature

```diff
@@ -113,6 +113,31 @@ Scenario: List all customers
     And I should see "user3" in the customer results
     And I should see "user4" in the customer results
 
+Scenario: Update all customer
+    When I visit the "Home Page"
+    And I set the "First Name" to "Tim"
+    And I set the "Last Name" to "Giovani"
+    And I press the "Search" button
+    Then I should see "Tim" in the "First Name" field
+    Then I should see "Giovani" in the "Last Name" field
+    And I should see "customer" in the "Category" field
+    When I change "First Name" to "Tina"
+    When I change "Last Name" to "Li"
```

Comment:
I think we should also include updating of the password and username fields and make sure those are changed too